src/cli/Merge.cpp

```diff
@@ -30,7 +30,7 @@ const QCommandLineOption Merge::SameCredentialsOption =
                        QObject::tr("Use the same credentials for both database files."));
 
 const QCommandLineOption Merge::KeyFileFromOption =
-    QCommandLineOption(QStringList() << "k"
```

Comment:
@wizzk42 I think we could also remove the short option and just keep `--key-file-from`. This would be consistent with `--yubikey-from`, which doesn't have a short version.

Reply:
✅  removed the short '-f' option
✅  updated the man page dropping the '-f'
✅  unit tests (for Merge.cpp cli options)